### ISSUE TITLE
'Submitting forms' example could use a better example

### ISSUE THREAD

**Issue by claudiodekker** [inertiajs/inertiajs.com#151]
On the https://inertiajs.com/forms#submitting-forms page, the first example is using a `form` object and a traditional Inertia post, without using the form helper. While this isn't bad per-se, I think it would be less confusing to adjust the example to reflect a situation in which you would use a traditional `Inertia.post` over the Inertia form.

**Comment by reinink** [comment]
Hmm, I'm not sure I agree. To me this is about teaching in steps. Before introducing the form helper, I think it's super helpful to understand how simple it is to handle forms manually in Inertia. This teaches them the core concepts, which they can then use in more complicated situations, when the form helper isn't enough. Put differently, I don't want to start with a more complicated example (nor do I want to start with a form helper example).

**Comment by claudiodekker** [comment]
Yeah, totally.. I'm just proposing an even simpler/basic example without the "form" word in it to prevent confusion

**Comment by reinink** [comment]
Yeah okay, maybe something like this?

```vue
<template>
  <form @submit.prevent="submit">
    <label for="name">Name:</label>
    <input id="name" v-model="name" type="text" />
    <label for="email">Email:</label>
    <input id="email" v-model="email" type="text" />
    <button type="submit">Submit</button>
  </form>
</template>

<script>
export default {
  data() {
    return {
      name: null,
      email: null,
    }
  },
  methods: {
    submit() {
      this.$inertia.post('/users', {
        name: this.name,
        email: this.email,
      })
    },
  },
}
</script>
```

**Comment by claudiodekker** [comment]
Yeah, exactly 👍 I think that'd do perfectly
